### PR TITLE
Featurebranch

### PR DESCRIPTION
**Pull request by yl642**
Add LDL HDL TOT analysis functions

### CODE REVIEW

File: chol_analysis.py [hunk 1]
```diff
@@ -1,10 +1,28 @@
-def HDL_analysis():
-    pass
-
-def LDL_analysis():
-    pass
-
-
+def HDL_analysis(HDL_level):
+    if HDL_level >= 60:
+        return 'Nromal'
```
Comment (dward2) [inline chol_analysis.py]: typo 'Nromal' should be 'Normal'

**Review by dward2 — CHANGES_REQUESTED** [review]
Normally, I don't need to review your pull requests.  But, I don't mind, either, if you want the practice and feedback of doing reviews.  Anyway, this code does not successfully execute due to an error.  Have you examined the error?

File: chol_analysis.py (hunk 1 above)
Comment (dward2) [reply chol_analysis.py]: By the way, this is not the error, this it is just me pointing out a different typo.

Comment (dward2) [reply chol_analysis.py]: I also just noticed that this is on my copy of the repository, not yours.  So, you can completely ignore this if you want.  I guess I didn't get enough sleep last night.

**Comment by yl642** [comment]
Sorry to bother~ but my first guess may be this error was due to that the two HDL analysis functions shared the same function name.

— Yangpei Liu
________________________________

**Comment by dward2** [comment]
> Sorry to bother~ but my first guess may be this error was due to that the two HDL analysis functions shared the same function name. — Yangpei Liu

No.  When I run the code, I get the following error:  
```
 File "D:/ClassRepos/BME547/mysandbox.py", line 32, in cholesterol_analysis
    HDL_analysis()
TypeError: HDL_analysis() missing 1 required positional argument: 'HDL_level'
```
Compare the call of the function to the definition of the function.

**Comment by yl642** [comment]
> > Sorry to bother~ but my first guess may be this error was due to that the two HDL analysis functions shared the same function name. — Yangpei Liu
> 
> No. When I run the code, I get the following error:
> 
> ```
>  File "D:/ClassRepos/BME547/mysandbox.py", line 32, in cholesterol_analysis
>     HDL_analysis()
> TypeError: HDL_analysis() missing 1 required positional argument: 'HDL_level'
> ```
> 
> Compare the call of the function to the definition of the function.

I fixed it! I forgot to input parameters when I was calling analysis functions.

**Review by dward2 — APPROVED** [review]

**Pull request merged**